### PR TITLE
chore: Formatting sweep (SC-303)

### PR DESCRIPTION
**Pull request by lucas-manuel**

### CODE REVIEW

File: src/VariableBorrowInterestRateStrategy.sol [hunk 1]
```diff
@@ -1,53 +1,65 @@
...
- * @title VariableBorrowInterestRateStrategy contract
- * @author Aave
- * @notice Implements the calculation of the interest rates depending on the reserve state
- * @dev The model of interest rate is based on 2 slopes, one before the `OPTIMAL_USAGE_RATIO`
- * point of usage and another from that one to 100%.
- * - An instance of this same contract, can't be used across different Aave markets, due to the caching
- *   of the PoolAddressesProvider
- * - Note: This is a modified version of DefaultReserveInterestRateStrategy with the stable borrow feature disabled.
+ * @title  VariableBorrowInterestRateStrategy contract
+ * @author SparkLend
```
Comment (hexonaut) [inline src/VariableBorrowInterestRateStrategy.sol]: Original author is Aave we can't change that, only add additional authors if we want, but it's fine.

File: src/VariableBorrowInterestRateStrategy.sol [hunk 2]
```diff
@@ -125,46 +147,66 @@ contract VariableBorrowInterestRateStrategy is IDefaultInterestRateStrategy {
...
         } else {
-            vars.currentVariableBorrowRate += _getVariableRateSlope1().rayMul(vars.borrowUsageRatio).rayDiv(
-                OPTIMAL_USAGE_RATIO
-            );
+            vars.currentVariableBorrowRate +=
+                _getVariableRateSlope1().rayMul(vars.borrowUsageRatio).rayDiv(OPTIMAL_USAGE_RATIO);
         }
 
+        // 4. Calculate the liquidity rate by multiplying the current variable borrow rate by
+        //    the supply usage ratio and subtracting the protocol's reserve factor. This yields the
```
Comment (hexonaut) [inline src/VariableBorrowInterestRateStrategy.sol]: The reserve factor is not extracted via subtract, but by multiplication.

**Review by hexonaut — CHANGES_REQUESTED** [review]
LGTM. 2 small changes.

File: src/VariableBorrowInterestRateStrategy.sol (hunk 1 above)
Comment (lucas-manuel) [reply src/VariableBorrowInterestRateStrategy.sol:17]: True updated

Comment (lucas-manuel) [reply src/VariableBorrowInterestRateStrategy.sol]: Fixed

File: src/VariableBorrowInterestRateStrategy.sol (hunk 2 above)
Comment (lucas-manuel) [reply src/VariableBorrowInterestRateStrategy.sol]: Fixed

**Review by hexonaut — APPROVED** [review]

**Pull request merged**